src/utils/node_byte_utils.ts

```diff
@@ -139,7 +139,7 @@ export const nodeJsByteUtils = {
       // TODO(NODE-4930): Insufficiently strict BSON UTF8 validation
       for (let i = 0; i < string.length; i++) {
         if (string.charCodeAt(i) === 0xfffd) {
-          if (!validateUtf8(buffer, start, end)) {
+          if (!validateUtf8(buffer, start, end, fatal)) {
```

Comment:
Because of the control flow around this we only ever call `validateUtf8` with true, which means it will throw so the if stmt doesn't make sense anymore. Can you clean that up and hard code true here instead of passing through fatal? just so it is clear that it is always on at this point.

Reply:
Yep, cleaned it up per our conversation